jest.config.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = {
+  testMatch: ['<rootDir>/tests/**/*.test.js'],
```

Comment:
The 'testMatch' pattern should be an array of glob patterns that Jest will use to detect test files. Ensure that the pattern provided matches the file structure of your tests. If your tests are located in a 'tests' folder and have the '.test.js' extension, the pattern should be '**/tests/**/*.test.js'.